### PR TITLE
[TEST] revisit DeviceViewProvider test

### PR DESCRIPTION
**Pull request by struss**
This commit will update getChildren test to cover more case.

ONE-vscode-DCO-1.0-Signed-off-by: struss <rrstrous@nate.com>

### CODE REVIEW

**Comment by struss** [comment]
this will increase `TEM` of `src/Execute/DeviceViewProvider.ts`

|Case|before|After|
|------|---|---|
|Statements| 81%(81/100)|91%(91/100)|
|Branch| 42.3%(11/26)|76.92%(20/26)|
|Function|88.88%(16/18)|88.88%(16/18)|
|Lines|80.61%(79/98)|90.81%(89/98)|

File: src/Tests/Execute/DeviceViewProvider.test.ts [hunk 1]
```diff
@@ -97,44 +99,65 @@ suite('DeviceViewProvider', function() {
...
+              assert.strictEqual(result[0].collapsibleState, vscode.TreeItemCollapsibleState.None);
+              assert.strictEqual(result[0].nodetype, NodeType.none);
+              assert.strictEqual(result[0].managerName, '');
+            }
+          }
+        }
+        done();
+      });
+    });
+    suite('#refresh', function() {
```
Comment (llFreetimell) [inline src/Tests/Execute/DeviceViewProvider.test.ts:156]: I think there is no test in this suite... was it intended?

Comment (struss) [reply src/Tests/Execute/DeviceViewProvider.test.ts:156]: Oh I will check this. thanks!

Comment (struss) [reply src/Tests/Execute/DeviceViewProvider.test.ts:156]: um....I think we don't need to check this. I forget to remove this. sorry for confusing.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**